### ISSUE TITLE
Deployment of D365FO LCS project and environments in France.

### ISSUE THREAD

**Issue by SasoGLIGOR** [MicrosoftDocs/dynamics-365-unified-operations-public#2043]
Hello,
We are trying to get our D365FO LCS project and environments in French datacenters since October 2019.
This seems not to be possible, we cannot deploy one single environment in France, not for the moment at least.
We have hard time getting answers from Microsoft also, do not know why this is not working and when will it work.
If anyone could give hand on this one, it would be great.
Thanks
Saso

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9be03020-97a8-0f3c-d39c-f50f9987a89d
* Version Independent ID: 0737be8a-48fc-3e5d-e99d-a5c4d54d0744
* Content: [Finance and Operations apps in France](https://docs.microsoft.com/en-us/dynamics365/fin-ops-core/dev-itpro/deployment/france-local-deployment?source=docs#feedback)
* Content Source: [articles/fin-ops-core/dev-itpro/deployment/france-local-deployment.md](https://github.com/MicrosoftDocs/Dynamics-365-Unified-Operations-Public/blob/live/articles/fin-ops-core/dev-itpro/deployment/france-local-deployment.md)
* Service: **dynamics-ax-platform**
* Product: ****
* Technology: ****
* GitHub Login: @tfehr
* Microsoft Alias: **tfehr**

**Comment by tonyafehr** [comment]
@SasoGLIGOR, have you contacted [Support](https://docs.microsoft.com/en-us/dynamics365/unified-operations/dev-itpro/lifecycle-services/lcs-support)? If not, please do so to get assistance from the Support team. If you have contacted Support, and you have a pending support ticket, please let us know so we can follow up in regard to this issue.

**Comment by SasoGLIGOR** [comment]
Hi Tonya and thanks for the reply.
Sure, we have contacted support, and still have an ongoing ticket with Microsoft.
The résumé is the following :
-	First we were asked to justify why do we need our environments in France.
-	Then, we needed to recreate our LCS project in the French version: fr.lcs.dynamics.com
-	Then, we weren’t able to complete the onboarding process, cannot choose Azure region.
-	Then, we weren’t able to deploy the environments, cannot choose Azure region, again.
-	Then, we deployed the environments but they stayed in “Queued” state.
-	Then, the Tier-1 environments were deployed, but not the Tier-2, incomplete state.
-	To be continued…
Here we are today, advancing but very slowly.
Objectively speaking, the possibility described in this article, is just not yet available.
Thanks
Saso

**Comment by tonyafehr** [comment]
Thanks for the update, I will forward your info on to the appropriate team. Please send me a personal email at tfehr@microsoft.com so I have your email address.

**Comment by SasoGLIGOR** [comment]
Hello,

I have some good news, we managed to deploy our D365FO environments in French DC.
Let me continue with the steps that followed my preceding post :
-	We had to redeploy all of the environments, the Azure connectors were incorrect.
-	Then, we were able to connect the Tier-2, but not to Tier-1 environments.
-	Then, we were able to connect to all of the environments, but the details page.
-	Then, after all of the tuning, we finally managed to have an operational LCS project.

There are some small side effects to be resolved, but this is really minor issues.
I hope that the path we traced will help French clients to get the service more effortlessly.

Thank you Tonya and MS support team.
Kind regards,
Saso
